sys/vane/gall.hoon

```diff
@@ -728,10 +728,16 @@
     ++  ap-fill                                         ::  add to queue
       ^-  {? _.}
       =+  suy=(fall (~(get by qel.ged) ost) 0)
-      ?:  =(20 suy)
-        [%| +]
-      ::  ~?  !=(20 suy)  [%ap-fill-add [[our dap] q.q.pry ost] +(suy)]
-      [%& +(qel.ged (~(put by qel.ged) ost +(suy)))]
+      =/  subscriber-ship  p:(~(got by sup.ged) ost)
+      ?:  &(=(20 suy) !=(our subscriber-ship))
```

Comment:
Huh, looks like you turned off the culling if our own ship is the one subscribed. Does that still work for web stuff?

Reply:
I tried a bunch of stuff to reduce the number of subscription updates in the queues between aqua/ph and aqua-*, and I couldn't come up with anything that would work reliably.

I'm not sure how web stuff uses this?  Is that the only way we clean up subscriptions created by web pages?